CHANGELOG.asciidoc

```diff
@@ -25,6 +25,8 @@ NEED AND IMAGE
 
 This release also includes changes from <<release-3-3-2, 3.3.2>>.
 
+* Fixed a bug in `ReducingBarrierStep`, that returned the provided seed value despite no elements being available.
```

Comment:
Should this be a changelog entry for 3.4.0 as this is pointed at the master branch?